### DISCUSSION TITLE
New UI

### DISCUSSION THREAD

**Discussion by HMarzban** [docs-plus/docs.plus#84] [General] [answered]
New google ui rollout, Lately. they are using [Material design v3](https://m3.material.io/).
It would be great to discuss the new UI and make a plan for the next steps to redesign the UI.

![image](https://user-images.githubusercontent.com/20157508/226839066-7c768d1e-f44e-4825-a96a-3cea5966a9d1.png)
![image](https://user-images.githubusercontent.com/20157508/226837841-418c16be-fd0a-470b-8d0d-7e32020ad803.png)

**Comment by edsaperia** [comment]
Ah yes, I saw this! Personally I don't like it, and I think we should stick with the current design...!

**Reply by edsaperia** [reply, marked as answer]
I'm a big skeuomorphism fan, specifically drop shadows. I think they are very helpful for communicating hierarchies.
